src/main/content/_i18n/en.yml

```diff
@@ -252,26 +252,26 @@ blog:
     part3: feed.
   tags:
     announcements: announcements
-    MicroProfile: MicroProfile
```

Comment:
I think we would want to change the right side value too, because that's what is shown on the site and the goal is to change the visible tags to lowercase.